langchain/src/embeddings/bedrock.ts

```diff
@@ -53,6 +55,7 @@ export class BedrockEmbeddings
       });
   }
 
+  /** @deprecated */
```

Comment:
Side question, could this method be deleted instead or would that not maintain backcompat?